### DISCUSSION TITLE
Cannot run Python code in Jupyter notebook in VScode

### DISCUSSION THREAD

**Discussion by abb174** [microsoft/vscode-jupyter#11983] [Questions and Answers]
I am receiving an error when trying to run a simple print Hello World script in python.  it says I require a ipykernel package. I installed python and jupyter extensions. Not sure what it is not running the code or what the kernel means. I have used VScode and Jupyterlab in the past. Even when I select to run the install for the kernel, nothing happens
![Screenshot 2022-11-10 135313](https://user-images.githubusercontent.com/82842951/201192893-98cc016c-6ee5-4b9f-a6fc-50bfb8633735.png)

**Comment by DonJayamanne** [comment]
Looks like the necessary dependencies arre missing, please could you run the command listed in the error message in your termina.
I.e. run the command `c:/.... -m pip install....` in your terminal.
